src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/designforextension/InputDesignForExtensionsLocalAnnotations.java

```diff
@@ -0,0 +1,12 @@
+package com.puppycrawl.tools.checkstyle.checks.design.designforextension;
+
+public class InputDesignForExtensionsLocalAnnotations
+{
+    public @interface Rule {
```

Comment:
@romani 
Have I understood you correctly that I shouldn't modify this file and all should be as is now?